mlx_embeddings/utils.py

```diff
@@ -52,6 +54,10 @@ def _get_classes(config: dict):
         logging.error(msg)
         raise ValueError(msg)
 
+    # Special handling for ColQwen2_5
```

Comment:
We can rename ColQwen2_5Config to ModelArgs and still pass TextConfig and VisionConfig because these exists in mlx_vlm unless there is a reason not to.